### PR TITLE
backport name cli arg to jobmanager

### PR DESCRIPTION
**Pull request by com4**

### CODE REVIEW

File: bin/emq-jobmanager [hunk 1]
```diff
@@ -48,6 +51,8 @@ if __name__ == "__main__":
 
     broker_addr = args.broker_addr
     concurrent_jobs = args.jobs
+    name = args.name
 
-    j = JobManager(queues=queues, concurrent_jobs=concurrent_jobs)
+    j = JobManager(queues=queues, concurrent_jobs=concurrent_jobs,
+                   name=name)
```
Comment (sideshowdave7) [inline bin/emq-jobmanager:57]: This will cause the name of the socket to be None if nothing is passed in on the command line.

**Review by sideshowdave7 — CHANGES_REQUESTED** [review]

File: bin/emq-jobmanager (hunk 1 above)
Comment (com4) [reply bin/emq-jobmanager:57]: the kwargs.pop will use the default if name is passed in as None

Comment (com4) [reply bin/emq-jobmanager:57]: that is a lie.

Comment (com4) [reply bin/emq-jobmanager:57]: https://github.com/eventmq/eventmq/pull/29/commits/4a10f8d27a1d55f81c0bfad7d15eece774998b99

**Pull request merged**